user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,461 @@
+.. _admin-capabilities:
+
+Capabilities List
+=================
+
+The *"Capabilities"* make an important part of the access and permission settings defined in the system, as described in 
+the :ref:`Access and Permissions Basics guide <user-guide-user-management-permissions-basic>`. 
+
+Depending on their specific roles, users will have access to different functionalities. For example, some users may not 
+be able to assign tags, share grids, or create new users.
+
+For each role and capability pair there are only two options:
+
+For each role and capability pair there are only two options:
+
+- **None**: users with the role won’t be able to use the functionality.
+- ***System***: users with the role will be able to use the functionality for all the records created within their 
+  OroCRM instance they’ve logged in into.
+
+  
+.. image:: ./img/roles/01.png   
+  
+This guide describes the set of capabilities available in the alphabetical order and provides examples of several 
+default capability settings for some common roles.
+
+
+
+.. _admin-capabilities-acc:
+
+Abandoned Cart Campaign manipulations
+-------------------------------------
+
+If this capability is set to *"System"*, users with the role can use the Abandoned Cart Campaign functionality to 
+generate automatic mailings to customers who have unfinished their purchase in your Magento store in order to convince 
+them to complete the checkout and place an order.  You can find more details about the functionality in the 
+:ref:`Magento Abandoned Cart Campaigns guide <user-guide-acc>`.
```

Comment:
Edit: If this capability is set to *"System,"* users with the role can use the Abandoned Cart Campaign functionality to generate automatic mailings to customers in your Magento store who have not completed their purchase in order to convince them to complete the checkout and place an order.  You can find more details about this functionality in the :ref:`Magento Abandoned Cart Campaigns guide <user-guide-acc>`.